src/components/state-details.js

```diff
@@ -230,6 +230,9 @@ export default function StateDetailsPage({ location, data }) {
   const currentSolarMWStr = numberToHumanString(currentSolarMW)
   const currentWindMWStr = numberToHumanString(currentWindMW)
 
+  // #### Number of Turbines and Football-fields of solar panels. ####
+  const turbinesNeededNumber = Math.ceil(targetGenByWindMW / 843 * 12)
```

Comment:
Where do 843 and 12 come from? This seems like a case of magic strings, I'd make those constants and cite a source

Reply:
@herrerarobert - pinging you on this - we should cite a source

Reply:
I have added the source in a comment.